info.femm.unofficial-manifest.yml

```diff
@@ -0,0 +1,235 @@
+
+
+# build wow64              
+  - name: org.winehq.wine_64bit-5.0.5
+    builddir: true
+    build-options:
+      make-args: -j8
+      prefix: /app
+      prepend-path: /app/bin
+      ldflags: -L/app/lib64 -L/app/lib -L/app/lib32
+      append-path: /usr/lib/sdk/toolchain-i386/bin
+      env:
+        CC:  i686-unknown-linux-gnu-gcc
+        CXX: i686-unknown-linux-gnu-g++
+      libdir: /app/lib32
+      strip: true
+      no-debuginfo: true
+      
+    make-install-args:
+      - LDCONFIG=/bin/true
+      - STRIP=/bin/true
+      - UPDATE_DESKTOP_DATABASE=/bin/true
+    buildsystem: simple
+    build-commands:
+    # 1. Build wine64
+    - export CC=x86_64-unknown-linux-gnu-gcc  &&
+      export CXX=x86_64-unknown-linux-gnu-g++ &&
+      mkdir wine64-build && cd wine64-build   &&
+      ../configure 
+      --prefix="/app" 
+      --enable-shared      
+      --disable-static
+      --disable-tests
+      --with-x
+      --without-cups
+      --without-curses
+      --without-capi
+      --without-gphoto
+      --without-gsm
+      --without-hal
+      --without-netapi
+      --enable-win64 &&  
+      make -j8
+
+    # 2. Build wine32 twice
+    - export CC=i686-unknown-linux-gnu-gcc  &&
+      export CXX=i686-unknown-linux-gnu-g++ &&
+      mkdir wine32-tools && cd wine32-tools &&
+      ../configure 
+      --prefix="/app" 
+      --enable-shared
+      --disable-static
+      --disable-tests
+      --with-x
+      --without-cups
+      --without-curses
+      --without-capi
+      --without-gphoto
+      --without-gsm
+      --without-hal
+      --without-netapi &&
+      make -j8
+
+    # 3. Build 32bit second time and 
+    # install 32 bit and 64 bit wine
+    - export CC=i686-unknown-linux-gnu-gcc  &&
+      export CXX=i686-unknown-linux-gnu-g++ &&
+      mkdir wine32-combo && cd wine32-combo &&
+      ../configure 
+      --with-wine64=../wine64-build
+      --with-wine-tools=../wine32-tools
+      --prefix="/app" 
+      --enable-shared      
+      --disable-static
+      --disable-tests
+      --with-x
+      --without-cups
+      --without-curses
+      --without-capi
+      --without-gphoto
+      --without-gsm
+      --without-hal
+      --without-netapi   &&
+      make -j8           && 
+      make install       && 
+      cd ../wine64-build && 
+      make install
+
+    sources:
+      - type: archive
+        url: https://dl.winehq.org/wine/source/5.0/wine-5.0.5.tar.xz
+        sha512: 3cf0ccb2eac0868f700296a62aeea9b856522ac63861beb49deed50136a334c450f2d1172f9d0195b4fe0bbf8f808f7350a79122726da1cbf5d21161f6dfca36
+        
+  - name: winetricks
+    buildsystem: simple
+    build-commands:
+    - make
+    - make PREFIX=/app install
+    - chmod +x /app/bin/winetricks
+    sources:
+    - type: git
+      url: https://github.com/Winetricks/winetricks.git
+      commit: 8b0eec06fef88fb91356e886d1ad3d554b37e6c9
+              
+  - name: femm-installer
+    buildsystem: simple
+    dir: ${WINEPREFIX}
```

Comment:
this property doesn't exist.

Reply:
Hi @hfiguiere, appreciate all your contribution. I'm not sure what I'm supposed to do here. Can you please give me some guidance?

Reply:
if I had a clue of what you are trying to do, but the property `dir` doesn't exist in that context. flatpak-builder should even spew a warning about it.